lib/gobstones/batch_parser.rb

```diff
@@ -43,9 +43,17 @@ def parse_options(test)
       [
         struct(key: :show_initial_board, default: true),
         struct(key: :show_final_board, default: true),
-        struct(key: :check_head_position, default: false),
+        struct(key: :check_head_position, default: true),
         struct(key: :subject, default: nil)
-      ].map { |it| [it.key, test[it.key] || it.default] }.to_h
+      ].map { |it| [
+          it.key,
+          if test[it.key].nil?
```

Comment:
Why this change? Can the key be `false`?

Reply:
This wasn't working well. For example, the initial value of `show_final_board` is true. If the YML had `false` in the YML, it was running `false || true` resulting in `true` again.

Reply:
OK, I see. It is a pity that there is a default whose value is `true` .

Reply:
In the future, i also recommend using a ternary if `...?...:...`